website/pages/docs/plugins/destinations/s3/overview.mdx

```diff
@@ -51,6 +51,10 @@ This is the (nested) spec used by the CSV destination Plugin.
 
   When `athena` is set to `true`, the S3 plugin will sanitize keys in JSON columns to be compatible with the Hive Metastore / Athena. This allows tables to be created with a Glue Crawler and then queried via Athena, without changes to the table schema.
 
+- `no_test` (boolean) (optional, default `false`)
```

Comment:
The double double negative of `no_test` and default `false` was a bit confusing to me... Would something like `test_write` with a default value of `true` be clearer?

Reply:
This was done to be the same as `no_rotate`, but since `no_rotate` has its own issues...

Reply:
Fixed in [`771348d` (#10287)](https://github.com/cloudquery/cloudquery/pull/10287/commits/771348d4862a4d5c9c504b32276f833d1ed5b96b)